mock matchers › failing output › expect.mock.nthCalledWith different value
<dim>expect.mock(</resetDimAndBold><red>mock</resetColor><dim>).nthCalledWith(</resetDimAndBold><green>expected</resetColor><dim>)</resetDimAndBold> <dim>/* using == */</resetDimAndBold>

Expected the mock function to have been called the second time with:
  <green>3</resetColor>
But it was called with:
  <red>2</resetColor>
